Headers/AppKit/NSSpeechSynthesizer.h

```diff
@@ -151,8 +151,13 @@ extern NSString *NSSpeechDictionaryEntryPhonemes;
 - (NSString *) phonemesFromText: (NSString *)text;
 @end
 
-// Speech synthesizer delegate informal protocol...
```

Comment:
I believe there is; it's lying if it says it's *always* an informal protocol, since if the compiler supports `@optional` it will be an actual protocol.